app/views/play/CampaignView.js

```diff
@@ -94,6 +94,7 @@ class CampaignView extends RootView {
       'click .roblox-level': 'onRobloxLevelClick',
       'click .hackstack-level': 'onHackStackLevelClick',
       'click .hackstack-menu-icon': 'onHackStackLevelClick',
+      'click .junior-menu-icon': 'openJuniorPromotionModal',
```

Comment:
can we add a tracking on how many clicked this one?

Reply:
![image](https://github.com/user-attachments/assets/e79ed3e5-c56c-4e14-af56-b1b49110ed0a)
actually this function has tracker, do we want to make another tracker?